src/rgw/rgw_lc.cc

```diff
@@ -1447,6 +1447,49 @@ class LCOpAction_Transition : public LCOpAction {
       return 0;
     }
 
+    /* notifications */
+    auto& bucket = oc.bucket;
+    auto& obj = oc.obj;
+
+    string etag;
+    r = obj->load_obj_state(oc.dpp, null_yield, true);
+    if (r < 0) {
+      ldpp_dout(oc.dpp, 0) <<
+	fmt::format("ERROR: get_obj_state() failed on transition of object k={} error r={}",
+		    oc.o.key.to_string(), r) << dendl;
+      return r;
```

Comment:
if we load the obj state only for the notification, why do we fail the operation here?